javascript-constructors/example-constructor.js

```diff
@@ -0,0 +1,7 @@
+function ExampleConstructor() {}
+console.log('Value of ExampleConstructor prototype: ', ExampleConstructor);
+console.log('Value of ExampleConstructor typeof: ', typeof ExampleConstructor);
```

Comment:
Actually we want to console log value and type of prototype property of ExampleConstructor